test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -0,0 +1,116 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <forward_list>
+#include <list>
+#include <type_traits>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/container/bitcompressed_vector.hpp>
+#include <seqan3/range/views/kmer_hash.hpp>
+#include <seqan3/range/views/minimiser.hpp>
+#include <seqan3/range/views/take_until.hpp>
+#include <seqan3/range/views/to.hpp>
+
+#include <gtest/gtest.h>
+
+using seqan3::operator""_dna4;
+using seqan3::operator""_shape;
+using result_t = std::vector<size_t>;
+
+static constexpr auto kmer_view = seqan3::views::kmer_hash(seqan3::ungapped{4});
+static constexpr auto gapped_kmer_view = seqan3::views::kmer_hash(0b1001_shape);
+static constexpr auto minimiser_view = seqan3::views::minimiser(5);
+static constexpr auto minimiser_view2 = seqan3::views::minimiser(1); // kmer_size == window_size, should throw
+
+template <typename T>
+class minimiser_view_properties_test: public ::testing::Test { };
+
+using underlying_range_types = ::testing::Types<std::vector<seqan3::dna4>,
+                                                std::vector<seqan3::dna4> const,
+                                                seqan3::bitcompressed_vector<seqan3::dna4>,
+                                                seqan3::bitcompressed_vector<seqan3::dna4> const,
+                                                std::list<seqan3::dna4>,
+                                                std::list<seqan3::dna4> const,
+                                                std::forward_list<seqan3::dna4>,
+                                                std::forward_list<seqan3::dna4> const>;
+
+TYPED_TEST_SUITE(minimiser_view_properties_test, underlying_range_types, );
+
+class minimiser_test : public ::testing::Test
+{
+protected:
+    std::vector<seqan3::dna4> text1{"AAAAAAAAAA"_dna4};
+    std::vector<seqan3::dna4> text1_short{"AAAAAA"_dna4};
+    result_t result1{0, 0, 0}; // Same result for ungapped and gapped
+    result_t result1a{0};      // windows_size == text_size, same result for ungapped and gapped
```

Comment:
```suggestion
    result_t result1_short{0};      // windows_size == text_size, same result for ungapped and gapped
```
or what does `a` mean?